@AH-64D Apache Longbow/Addons/fza_ah64_mpd/functions/fn_init.sqf

```diff
@@ -8,6 +8,7 @@ _heli setVariable ["fza_mpd_tsdShowNavWptData", true];
 _heli setVariable ["fza_mpd_tsdShowAtkCurrRoute", false];
 _heli setVariable ["fza_mpd_tsdShowEnemy", [false, false]];
 _heli setVariable ["fza_mpd_tsdShowFriendly", [false, true]];
+_heli setVariable ["fza_mpd_tsdShowRlwr", [true, true]];
```

Comment:
Not required for this PR, but all of these should probably be moved to `_persistState`